### PR TITLE
Add instructions and YAML for Prometheus scraping

### PR DESCRIPTION
**Pull request by igable**
Add an example Prometheus scrape job configuration and provide
instructions for configuring Prometheus to scrape stardog pods

### CODE REVIEW

**Comment by igable** [comment]
I'm sure I'm going to find some typos on monday, but this is the general direction I'm headed.

File: examples/grafana/README.md [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+
+Step 1: Configuring Prometheus to scrape Stardog
+------------------------------------------------
+
+We recommend that you create a Stardog user specifically for monitoring. This user will not be
+able to read data but will be able to access the Prometheus monitoring endpoint.
+
+    stardog-admin user add monitor
+    stardog-admin role add monitoring
+    stardog-admin user addrole --role monitoring monitor
```
Comment (buzztroll) [inline examples/grafana/README.md]: is the role needed?

File: examples/grafana/README.md [hunk 2]
```diff
@@ -1,14 +1,59 @@
-# Grafana Dashboards - Stardog Metrics
+Stardog Grafana Dashboard
+=========================
+
+Stardog provides a preconfigured Stardog dashboard for Grafana in the form of a json files which can
+be imported into Grafana. In order to be able to use the dashboard you must have Prometheus
+configured to scrape Stardog kubenetes pods for metrics data.
+
+*screen shots go here*
```
Comment (buzztroll) [inline examples/grafana/README.md:8]: should we get these in before merging?

**Review by buzztroll — CHANGES_REQUESTED** [review]

File: examples/grafana/README.md (hunk 1 above)
Comment (igable) [reply examples/grafana/README.md]: discussed in slack. Dropping role creation.

File: examples/grafana/README.md (hunk 2 above)
Comment (igable) [reply examples/grafana/README.md:8]: We are merging to a feature branch and not develop so that Simon can add the screen shots.

Comment (buzztroll) [reply examples/grafana/README.md:8]: 👍

**Review by buzztroll — APPROVED** [review]

**Pull request merged**